fieldservice/views/res_partner.xml

```diff
@@ -0,0 +1,103 @@
+<odoo>
+
+    <!-- Customers -->
+    <record id="action_fsm_customer" model="ir.actions.act_window">
+        <field name="name">Customers</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_mode">kanban,tree,form</field>
+        <field name="domain">[('customer', '=', True)]</field>
```

Comment:
```suggestion
        <field name="domain">[('customer_rank','>', 0)]</field>
```
Looks like something new in v13.

Receiving error:
`ValueError: Invalid field 'customer' in leaf "<osv.ExtendedLeaf: ('customer', '=', True) on res_partner (ctx: )>"`